al1.pkr.hcl

```diff
@@ -136,6 +136,13 @@ build {
     ]
   }
 
+  provisioner "shell" {
+    inline_shebang = "/bin/sh -ex"
+    inline = [
+      "sudo yum update -y --security --sec-severity=critical --exclude=nvidia*,docker*,cuda*,containerd*"
```

Comment:
Do we want to exclude `runc`?

Reply:
I've only excluded values that are also excluded in the check-update-security.sh script, since that might trigger a release, we want to update the same things that caused the release trigger. Also, runc doesn't seem to be a dependency in the variable.pkr.hcl file for al1 and al2, so it was never added to the pinned dependencies within the readjusted release-{ami-type}.pkrvrs.hcl file